spec/support/serializers.rb

```diff
@@ -1,4 +1,20 @@
 module MyApp
+  class PostWithContext
```

Comment:
Please move these down to the end of the module, so things continue to be ordered a bit more logically.